### PR TITLE
README - add info on available snapshot CSV files in releases area

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Partially address info discussed in https://github.com/scikit-hep/scikit-hep-orgstats/issues/3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,9 +61,13 @@ Commands:
   main  Make a weekly or daily downloads plot
 ```
 
-You can run multiple commands and often give options multiple times. `main`, `freq`, and `all`, support `--key KEY`, thought they choose a nice default. The two "per-week" plots also support a `--daily` flag to change weekly into daily statistics.
+You can run multiple commands and often give options multiple times.
+`main`, `freq`, and `all`, support `--key KEY`, though they choose a nice default.
```
Comment (eduardo-rodrigues) [inline README.md:65]: @henryiii, I think this is what you meant to say, but correct me otherwise.

File: README.md [hunk 2]
```diff
@@ -24,15 +24,19 @@ You will have to set up your credentials as described [here](https://cloud.googl
 
 ## Download
 
-Then, you can run the download script:
+Then, you can run the download script
```
Comment (henryiii) [inline README.md]: The colon is more standard. Up to the point that restructured text literally is harder to write without it.

File: README.md (hunk 1 above)
Comment (henryiii) [reply README.md:65]: Yes.

**Review by henryiii — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (eduardo-rodrigues) [reply README.md]: OK, reverting back.

**Pull request merged**